docs/docs/features/backlight.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Backlight
+sidebar_label: Backlight
+---
+
+Backlight is a feature used to control array of LEDs, usually placed through or under switches. Unlike [RGB Underglow](underglow.md), backlight currently allows only one color per LED, also LEDs are not addressable, so you can't control individual LEDs.
+
+## Enabling Backlight
+
+To enable backlight on your board or shield, add the following lines to your `.conf` file of your user config directory as such:
+
+```
+CONFIG_PWM=y
+CONFIG_LED_PWM=y
+CONFIG_ZMK_BACKLIGHT=y
+```
+
+If your board or shield does not have backlight configured, refer to [Adding Backlight to a Board](#adding-backlight-to-a-board).
+
+## Configuring Backlight
+
+There are various Kconfig options used to configure the backlight feature. These can all be set in the `.conf` file.
+
+| Option                               | Description                                           | Default |
+| ------------------------------------ | ----------------------------------------------------- | ------- |
+| `CONFIG_ZMK_BACKLIGHT_BRT_STEP`      | Brightness step in percent                            | 20      |
+| `CONFIG_ZMK_BACKLIGHT_BRT_START`     | Default brightness in percent                         | 40      |
+| `CONFIG_ZMK_BACKLIGHT_ON_START`      | Default backlight state                               | y       |
+| `CONFIG_ZMK_BACKLIGHT_AUTO_OFF_IDLE` | Turn off backlight when keyboard goes into idle state | y       |
+| `CONFIG_ZMK_BACKLIGHT_AUTO_OFF_USB`  | Turn off backlight when USB is disconnected           | n       |
+
+## Adding Backlight to a Board
+
+Backlight is always added to a board, not a shield.
+If you have a shield with backlight, you must add a `boards/` directory within your shield folder to define the backlight individually for each board that supports the shield.
+Inside the `boards/` folder, you define a `<board>.overlay` for each different board.
+
+First, you need to enable PWM by adding the following lines to your `.overlay` file:
+
+```
+&pwm0 {
+	status = "okay";
+	ch0-pin = <45>;
+	/* ch0-inverted; */
+};
+```
+
+The value `ch0-pin` represents the pin that controls the LEDs. To calculate the value to use, you need a bit of math. You need the hardware port and run it through a function.
```

Comment:
We should note this is nRF52 specific, and if this is for a shield, this should go in a board specific file in `<shield>/boards/nice_nano_v2.overlay`, for instance.

Reply:
Added a note about nRF52, there was already a note about boards and shields